test/import-test.js

```diff
@@ -311,8 +311,8 @@ describe('#importDataElement', () => {
     expectValue(entry.value, 'codeConstraintOnValueChildOut', 'CodedFromValueSet');
     expect(entry.value.constraints).to.have.length(1);
     expect(entry.value.constraints[0]).to.be.instanceof(CodeConstraint);
-  console.log("Test 20: entry.value.constraints[0] = "+ JSON.stringify(entry.value.constraints[0]));  // missing path
-    expect(entry.value.constraints[0].path).to.eql([id('codeConstraintOnValueChildOut','CodedFromValueSet')]);   // fails here because the path is missing
+    console.log("Test 20: entry.value.constraints[0] = "+ JSON.stringify(entry.value.constraints[0]));
```

Comment:
Did you mean to leave this `console.log` as-is (_not_ commented out)?

Reply:
yeah those were left in by Mark for debugging purposes.